libs/glib2/patches/003-valgrind.h-mips16-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/glib/valgrind.h	2019-12-12 14:53:26.000200499 +0100
++++ b/glib/valgrind.h	2019-12-12 14:49:45.056163300 +0100
+@@ -157,7 +157,7 @@
+ #  define PLAT_s390x_linux 1
+ #elif defined(__linux__) && defined(__mips__) && (__mips==64)
+ #  define PLAT_mips64_linux 1
+-#elif defined(__linux__) && defined(__mips__) && (__mips!=64)
++#elif defined(__linux__) && defined(__mips__) && (__mips!=64) && !defined(__mips16)
```

Comment:
this can be simplified by replacing __mips!=64 with __mips==32 . It's more consistent as well.

Reply:
sure include this in your new pr, i will remove the glib changes from this PR and will reference yours.